### DISCUSSION TITLE
Auswertung übereinander legen

### DISCUSSION THREAD

**Discussion by Deanw1975** [solectrus/solectrus#2109] [Ideas]
Hallo zusammen,

nachdem ich einen Hinweis bekommen haben das nun Solectrus auch EVVC versteht (MQTT) habe ich mich gestern dran gemacht und einen Raspi aus der Schublade gezogen.

Nach der Installation war ich echt begeistert und danke für die Arbeit.

Eines würde mich noch freuen.
Kann man irgendwie die einzelnen Grafiken übereinander legen.
So das man erkennen kann, wohin der Strom über den Tag, der Woche, des Monats oder des Jahres geflossen ist.
Ich mache das gerade mit Fhem schon für den Tag:
![Fhem_Grafik](https://github.com/solectrus/solectrus/assets/108386811/8222dea3-8b15-4dbf-a8ec-233a5ee4dee9)

Wenn das da wäre, dann könnte ich meiner Frau endlich den Zugriff auf diese Übersichts-Werte geben.
;-)

Danke
VG
Dean

**Comment by StevieC121176** [comment]
Hatte ich hier #2056 schon ein Feature Request eröffnet.

**Reply by Deanw1975** [reply]
Danke, hatte ich nicht gefunden:-(
Dann kann dieser geschlossen werden und ich unterstützte gerne den anderen Beitrag!
